R/liftover.R

```diff
@@ -0,0 +1,287 @@
+#' @title Liftover Bedpe.
+#'
+#' @description Use liftOver to convert a bedpe file between the two main genome builds (grch37/hg38).
+#'
+#' @details The user can specify a path to the bedpe file that needs to be lifted with `bedpe_file`,
+#' or, the user can specify the bedpe data in a data frame with `bedpe_df`.
+#' The other required parameter is `target_build`, this parameter decides the final projection of the lifted data.
+#'
+
+#' @param data_df Input data to be lifted to another projection in the format of data frame.
+#' @param mode Specify the type of data to be converted. Available options are "bedpe" (default), "maf", and "bed".
```

Comment:
Do we want to add CN data in the future?

Reply:
CN is basically same as bed format so it is already handled. But when looking into this I found that since we do fill of the empty segments there are many mismatched regions between the 2 builds so it creates a lot of mismatched pairs after lifting (the downside of liftover itself, not related to this function) and that is why I choose the ashm sites to demonstrate the bed functionality